app/src/main/java/io/neurolab/main/ProgramModeActivity.java

```diff
@@ -64,9 +64,8 @@ protected void onCreate(Bundle savedInstanceState) {
                 break;
             case MEMORY_GRAPH_MODE:
                 setTitle(R.string.mem_graph);
-                break;
-            case SERIAL_PROGRAM_MODE:
```

Comment:
If you are replacing the serial program mode with memory graph mode, remove/replace the related things to 'serial mode' everywhere (in strings.xml for example and also other places) else they clutter up unnecessarily.

Reply:
Done. Thanks.